### PR TITLE
asynchronous data fetching from local storage (week7)

### PR DESCRIPTION
**Pull request by psharma1984**

### CODE REVIEW

**Review by JoHeyJo — CHANGES_REQUESTED** [review]
Are you able to run your app locally after clearing local storage? There is a bug that's preventing your app from running.

**Comment by JoHeyJo** [comment]
Let's also provide a "description" for your PRs.

**Comment by psharma1984** [comment]
@JoHeyJo pushed the changes. i cleared the local storage and its running locally now

File: src/App.js [hunk 1]
```diff
@@ -2,16 +2,34 @@ import React, { useEffect, useState } from 'react';
 import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
```
Comment (JoHeyJo) [inline src/App.js:4]: Remove extra space between modules and component declaration

File: src/App.js [hunk 2]
```diff
@@ -2,16 +2,34 @@ import React, { useEffect, useState } from 'react';
...
-    localStorage.setItem('savedTodoList', JSON.stringify(todoList))
-  }, [todoList])
-  return [todoList, setTodoList]
-}
+
 
 function App() {
-  const [todoList, setTodoList] = useSemiPersistentState();
+  const [todoList, setTodoList] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
```
Comment (JoHeyJo) [inline src/App.js:7]: It's good practice to have a space between functions and state.

**Review by JoHeyJo — APPROVED** [review]
Nicely done!  A few really minor "cosmetic issues" that will help with legibility and then you're ready to merge.

Resubmit so I can approve on CTD.  Great job!

File: src/App.js (hunk 1 above)
Comment (psharma1984) [reply src/App.js:4]: Sure

**Pull request merged**